CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 #### Added
 - Add `onlyCopyFilesOnInstall` option to targets for the Embed Files build phase. [#912](https://github.com/yonaskolb/XcodeGen/pull/912) @jsorge
+- Adds App Clip support. [#909](https://github.com/yonaskolb/XcodeGen/pull/909) @brentleyjones
```

Comment:
@brentleyjones Can you tag me in on this as well? 😄

Reply:
Yep!